modules/profile/files/jenkins/master/configure_git_user.groovy

```diff
@@ -0,0 +1,12 @@
+/* Configure Git User */
+
+/* This script is designed to be run via jenkins-cli.jar in order to update the
+ * configuration for the Jenkins GitSCM plugin.
+ */
+import jenkins.model.Jenkins
+
+def gitscm_config = jenkins.model.Jenkins.getInstance().getDescriptor("hudson.plugins.git.GitSCM")
+gitscm_config.setCreateAccountBasedOnEmail(false)
+gitscm_config.setGlobalConfigName("jenkins")
+gitscm_config.setGlobalConfigEmail("jenkins@build.ros.org")
+gitscm_config.save()
```

Comment:
I didn't bother checking for a change in this script since it's quite simple. In other groovy scripts I can envision checking current state before mutating and if the jenkins default is in place, replace it with our desired state but otherwise leave it be.